cfy_manager/components/manager/manager.py

```diff
@@ -110,6 +112,21 @@ def _set_selinux_enforcing(self):
         else:
             logger.debug('SELinux is enforced.')
 
+    def _set_selinux_policies(self):
+        # Setup Cloudify SELinux policies for Manager
+        selinux_state = self._get_selinux_state()
+        if selinux_state == 'Disabled':
```

Comment:
I think we should probably deploy the policy anyway- that way everything will keep working if the user enables selinux later.